### PR TITLE
4.4.0 images update - Adjust SEO - Part 2

### PR DESCRIPTION
**Pull request by Arcapas**
## Description
This PR aims to adjust the images' SEO details. 

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/deployment-options/splunk/splunk-app.rst [hunk 1]
```diff
@@ -139,9 +147,11 @@ Install the Wazuh app for Splunk in a minimal distributed architecture
 
 #. The app will redirect you to the Settings tab. If you are not redirected to the Settings tab, select the settings icon, this will take you to the “Settings” page.
 
-   .. thumbnail:: /images/splunk-app/16.png
+   .. thumbnail:: /images/splunk-app/settings-tab.png
```
Comment (javimed) [inline source/deployment-options/splunk/splunk-app.rst:150]: This is more the Settings "icon" and not the Settings tab

File: source/deployment-options/splunk/splunk-app.rst [hunk 2]
```diff
@@ -151,9 +161,11 @@ Install the Wazuh app for Splunk in a minimal distributed architecture
     
       You can get more information about how to set up the credentials at :doc:`Securing the Wazuh API </user-manual/api/securing-api>` section.
 
-   .. thumbnail:: /images/splunk-app/17.png
+   .. thumbnail:: /images/splunk-app/wazuh-api-credentials.png
```
Comment (javimed) [inline source/deployment-options/splunk/splunk-app.rst:164]: This image looks very similar to the `wazuh-api-configuration.png`

File: source/deployment-options/splunk/splunk-app.rst [hunk 3]
```diff
@@ -163,17 +175,21 @@ Install the Wazuh app for Splunk in a minimal distributed architecture
...
       :width: 100%
+      :title: Settings/Index
+      :alt: Settings/Index
         
    You can also select the API, Index, and Source Type using the Quick settings menu.
     
 #. Open the “Overview” tab, and you should start seeing alerts and events.
 
-   .. thumbnail:: /images/splunk-app/19.png
+   .. thumbnail:: /images/splunk-app/alerts-and-events.png
```
Comment (javimed) [inline source/deployment-options/splunk/splunk-app.rst:188]: This one seems more like an Alerts dashboard

File: source/deployment-options/splunk/splunk-app.rst [hunk 4]
```diff
@@ -276,17 +296,21 @@ Install the Wazuh app for Splunk on the ``deployer`` machine and follow the step
...
       :width: 100%
+      :title: Index tab
+      :alt: Index tab
     
    You can also select the API, Index, and Source Type using the Quick settings menu.
 
 #. Open the “Overview” tab, and you should start seeing alerts and events.
 
-   .. thumbnail:: /images/splunk-app/23.png
+   .. thumbnail:: /images/splunk-app/alerts-and-events-tab.png
```
Comment (javimed) [inline source/deployment-options/splunk/splunk-app.rst:309]: Looks like an alerts dashboard

File: source/deployment-options/splunk/splunk-minimal-distributed.rst [hunk 5]
```diff
@@ -15,7 +15,9 @@ These are the two main components in this type of multi-tier server:
 -  The **indexer** runs the Splunk engine and is run on a different server from the Wazuh server. It reads forwarded data, parses, indexes and stores it as events that contain alert data generated by the Wazuh manager and sent by the Forwarder instance.
 -  The **forwarder** runs on the Wazuh manager and the Wazuh API instance. It reads local data and sends it to the indexer.
 
-.. thumbnail:: /images/splunk-cluster/4.png
+.. thumbnail:: /images/splunk-cluster/splunk-distributed-architecture.png
+   :title: Splunk distributed architecture
```
Comment (javimed) [inline source/deployment-options/splunk/splunk-minimal-distributed.rst:19]: This is the minimal distributed architecture

File: source/getting-started/use-cases/cloud-security.rst [hunk 6]
```diff
@@ -97,8 +97,9 @@ See below an example of an alert generated when an AWS security group is deleted
 
 Example of AWS dashboard:
 
-.. thumbnail:: /images/getting-started/use-cases/wazuh-use-cases-cloud-geo.png
-   :title: AWS dashboard
+.. thumbnail:: /images/getting-started/use-cases/aws-security.png
```
Comment (javimed) [inline source/getting-started/use-cases/cloud-security.rst:100]: This is AWS security dashboard

File: source/getting-started/use-cases/configuration-assessment.rst [hunk 7]
```diff
@@ -68,18 +68,21 @@ macOS SCA rule example:
 
 Below is an example of the results of a configuration assessment evaluation. These can be obtained through the web user interface or directly through the Wazuh RESTful API.
 
-.. thumbnail:: /images/getting-started/use-cases/wazuh-use-cases-sca01.png
-   :title: Security configuration assessment inventory dashboard
+.. thumbnail:: /images/getting-started/use-cases/security-configuration-assessment.png
```
Comment (javimed) [inline source/getting-started/use-cases/configuration-assessment.rst:71]: This the available  policies for the inventory

File: source/getting-started/use-cases/configuration-assessment.rst [hunk 8]
```diff
@@ -68,18 +68,21 @@ macOS SCA rule example:
...
+   :title: Security configuration assessment
+   :alt: Security configuration assessment
    :align: center
    :width: 80%
 
-.. thumbnail:: /images/getting-started/use-cases/wazuh-use-cases-sca2.png
+.. thumbnail:: /images/getting-started/use-cases/security-configuration-assessment-inventory.png
```
Comment (javimed) [inline source/getting-started/use-cases/configuration-assessment.rst:77]: This is the scan results

File: source/getting-started/use-cases/configuration-assessment.rst [hunk 9]
```diff
@@ -68,18 +68,21 @@ macOS SCA rule example:
...
    :title: Security configuration assessment inventory
+   :alt: Security configuration assessment inventory
    :align: center
    :width: 80%
 
-.. thumbnail:: /images/getting-started/use-cases/wazuh-use-cases-sca3.png
+.. thumbnail:: /images/getting-started/use-cases/security-configuration-assessment-events.png
    :title: Security configuration assessment inventory events
+   :alt: Security configuration assessment inventory events
```
Comment (javimed) [inline source/getting-started/use-cases/configuration-assessment.rst:85]: This is not hte inventory but the events tab

File: source/installation-guide/wazuh-agent/wazuh-agent-package-macos.rst [hunk 10]
```diff
@@ -44,8 +44,10 @@ The agent runs on the endpoint you want to monitor and communicates with the Waz
 
              #. To install the Wazuh agent on your system, run the downloaded file and follow the steps in the installation wizard. If you are not sure how to answer some of the prompts, use the default answers.
 
-                .. thumbnail:: ../../images/installation/macos.png
+                .. thumbnail:: ../../images/installation/macos-agent.png
                    :align: center
+                   :title: macOS agent
```
Comment (javimed) [inline source/installation-guide/wazuh-agent/wazuh-agent-package-macos.rst:49]:
```suggestion
                   :title: macOS agent installer
```

File: source/installation-guide/wazuh-agent/wazuh-agent-package-macos.rst [hunk 11]
```diff
@@ -44,8 +44,10 @@ The agent runs on the endpoint you want to monitor and communicates with the Waz
...
+                   :alt: macOS agent
```
Comment (javimed) [inline source/installation-guide/wazuh-agent/wazuh-agent-package-macos.rst:50]:
```suggestion
                   :alt: macOS agent installer
```

File: source/installation-guide/wazuh-agent/wazuh-agent-package-windows.rst [hunk 12]
```diff
@@ -54,9 +54,11 @@ The agent runs on the endpoint you want to monitor and communicates with the Waz
 
                 To install the Wazuh agent on your system, run the Windows installer and follow the steps in the installation wizard. If you are not sure how to answer some of the prompts, use the default answers. Once installed, the agent uses a GUI for configuration, opening the log file, and starting or stopping the service.
 
-                    .. thumbnail:: ../../images/installation/windows-agent.png
+                    .. thumbnail:: ../../images/installation/windows-agent.png                        
                         :align: center
                         :width: 100%
+                        :title: Windows agent
```
Comment (javimed) [inline source/installation-guide/wazuh-agent/wazuh-agent-package-windows.rst:60]:
```suggestion
                        :title: Windows agent manager
```

File: source/installation-guide/wazuh-agent/wazuh-agent-package-windows.rst [hunk 13]
```diff
@@ -54,9 +54,11 @@ The agent runs on the endpoint you want to monitor and communicates with the Waz
...
+                        :alt: Windows agent
```
Comment (javimed) [inline source/installation-guide/wazuh-agent/wazuh-agent-package-windows.rst:61]:
```suggestion
                        :alt: Windows agent manager
```

File: source/release-notes/release-4-3-0.rst [hunk 14]
```diff
@@ -42,9 +42,10 @@ An installation assistant ``wazuh-install.sh`` is available to users, allowing a
 
 Now, the agent is able to collect the installed packages inventory on Amazon Linux and Arch Linux, giving support to Vulnerability Detector for reporting vulnerability exposures. In addition, the Vulnerability Detector now manages a vulnerability inventory and produces alerts during the first agents scan and when a new vulnerability is either found or solved.
 
-.. thumbnail::  ../images/release-notes/4.3.0/packages-inventory.png
-      :title: Packages inventory
-      :align: center
+.. thumbnail:: /images/release-notes/4.3.0/vulnerability-detection.png
+      :title: vulnerability detection
```
Comment (javimed) [inline source/release-notes/release-4-3-0.rst:46]:
```suggestion
      :title: vulnerability detection inventory
```

File: source/release-notes/release-4-3-0.rst [hunk 15]
```diff
@@ -42,9 +42,10 @@ An installation assistant ``wazuh-install.sh`` is available to users, allowing a
...
+      :alt: vulnerability detection
```
Comment (javimed) [inline source/release-notes/release-4-3-0.rst:47]:
```suggestion
      :alt: vulnerability detection inventory
```

File: source/upgrade-guide/wazuh-agent/macos.rst [hunk 16]
```diff
@@ -30,9 +30,10 @@ Follow these steps to upgrade Wazuh agents locally on macOS systems. If you want
 
          Using the GUI will perform a simple upgrade. Double click on the downloaded file and follow the wizard. If you are not sure how to answer some of the prompts, simply use the default answers.
 
-         .. image:: /images/installation/macos.png
+         .. image:: /images/installation/macos-agent.png
             :align: center
             :scale: 50 %
+
```
Comment (javimed) [inline source/upgrade-guide/wazuh-agent/macos.rst:36]: title and alt for macOS agent installer are missing here

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/deployment-options/splunk/splunk-app.rst (hunk 1 above)
Comment (Arcapas) [reply source/deployment-options/splunk/splunk-app.rst:150]: You are right, and I'll change it!

File: source/deployment-options/splunk/splunk-app.rst (hunk 3 above)
Comment (Arcapas) [reply source/deployment-options/splunk/splunk-app.rst:188]: You are right, and I'll change it!

File: source/deployment-options/splunk/splunk-app.rst (hunk 4 above)
Comment (Arcapas) [reply source/deployment-options/splunk/splunk-app.rst:309]: You are right, and I'll change it!

File: source/getting-started/use-cases/cloud-security.rst (hunk 6 above)
Comment (Arcapas) [reply source/getting-started/use-cases/cloud-security.rst:100]: You are right, and I'll change it!

File: source/getting-started/use-cases/configuration-assessment.rst (hunk 7 above)
Comment (Arcapas) [reply source/getting-started/use-cases/configuration-assessment.rst:71]: Changes

File: source/getting-started/use-cases/configuration-assessment.rst (hunk 8 above)
Comment (Arcapas) [reply source/getting-started/use-cases/configuration-assessment.rst:77]: Changes

File: source/getting-started/use-cases/configuration-assessment.rst (hunk 9 above)
Comment (Arcapas) [reply source/getting-started/use-cases/configuration-assessment.rst:85]: Changes

File: source/upgrade-guide/wazuh-agent/macos.rst (hunk 16 above)
Comment (Arcapas) [reply source/upgrade-guide/wazuh-agent/macos.rst]: Added

**Review by javimed — APPROVED** [review]

**Pull request merged**